chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/reporting/DefaultSessionStatusDecisionMaker.java

```diff
@@ -31,24 +30,12 @@
 
 public class DefaultSessionStatusDecisionMaker implements SessionStatusDecisionMaker {
     private static final Logger log = LoggerFactory.getLogger(DefaultSessionStatusDecisionMaker.class);
-    private final double EPSILON = 0.0000001;
     private double successRateThreshold = 1.0;
```

Comment:
remove as well is is not used

\+ remove usage of setter for this parameter and corresponding property: chassis.master.reporting.successrate.threshold=1.0